plugins/woocommerce-admin/client/marketing/overview-multichannel/Campaigns/CreateNewCampaignModal/CreateNewCampaignModal.tsx

```diff
@@ -0,0 +1,164 @@
+	const { data: recommendedChannels } = useRecommendedChannels();
+	const { refetch } = useRegisteredChannels();
+
+	return (
+		<Modal
+			className={ classnames(
+				className,
+				'woocommerce-marketing-create-campaign-modal'
+			) }
+			title={ __( 'Create a new campaign', 'woocommerce' ) }
+			{ ...restProps }
+		>
+			<div className="woocommerce-marketing-new-campaigns">
+				<div className="woocommerce-marketing-new-campaigns__question-label">
+					{ __(
+						'Where would you like to promote your products?',
+						'woocommerce'
+					) }
+				</div>
+				<div>
+					{ newCampaignTypes?.map( ( el ) => {
+						return (
+							<Flex
+								key={ el.id }
+								className="woocommerce-marketing-new-campaign-type"
+								gap={ 4 }
+							>
+								<FlexItem>
+									<img
+										src={ el.icon }
+										alt={ el.name }
+										width="32"
+										height="32"
+									/>
+								</FlexItem>
+								<FlexBlock>
+									<Flex direction="column" gap={ 1 }>
+										<FlexItem className="woocommerce-marketing-new-campaign-type__name">
+											{ el.name }
+										</FlexItem>
+										<FlexItem className="woocommerce-marketing-new-campaign-type__description">
+											{ el.description }
+										</FlexItem>
+									</Flex>
+								</FlexBlock>
+								<FlexItem>
+									<Button
+										variant="secondary"
+										href={ el.createUrl }
+										target={
+											isExternalURL( el.createUrl )
+												? '_blank'
+												: '_self'
+										}
+									>
+										<Flex gap={ 1 }>
+											<FlexItem>
+												{ __(
+													'Create',
+													'woocommerce'
+												) }
+											</FlexItem>
+											{ isExternalURL( el.createUrl ) && (
+												<FlexItem>
+													<Icon
+														icon={ external }
+														size={ 16 }
+													/>
+												</FlexItem>
+											) }
+										</Flex>
+									</Button>
+								</FlexItem>
+							</Flex>
+						);
+					} ) }
+				</div>
+			</div>
+			{ !! recommendedChannels?.length && (
+				<div className="woocommerce-marketing-add-channels">
+					<Flex direction="column">
+						<FlexItem>
+							<Button
+								variant="link"
+								onClick={ () => setCollapsed( ! collapsed ) }
+							>
+								{ __(
+									'Add channels for other campaign types',
+									'woocommerce'
+								) }
+								<Icon
+									icon={ collapsed ? chevronDown : chevronUp }
+									size={ 24 }
+								/>
+							</Button>
+						</FlexItem>
+						{ ! collapsed && (
+							<FlexItem>
+								{ recommendedChannels.map( ( el, idx ) => {
+									return (
+										<Fragment key={ el.plugin }>
+											<SmartPluginCardBody
+												plugin={ el }
+												onInstalledAndActivated={
+													refetch
+												}
+											/>
+											{ idx !==
+												recommendedChannels.length -
+													1 && <CardDivider /> }
```

Comment:
😂  That's because I was doing copy and paste from the Channels component. I think at one point it was a shared component (or maybe I wanted to make it a shared component), so there was a CardDivider and I hid it by using CSS. Thanks for catching this. I have removed them in [`e15305b` (#37044)](https://github.com/woocommerce/woocommerce/pull/37044/commits/e15305b7c3db4e3c6b650bd71002faa82e18aec5).